sdk/tracing/azure-core-tracing-opencensus/src/main/java/com/azure/core/tracing/opencensus/OpenCensusTracer.java

```diff
@@ -25,28 +25,28 @@
 import static io.opencensus.trace.Link.Type.PARENT_LINKED_SPAN;
 
 /**
- * OpenTelemetry span
+ * OpenCensus span
```

Comment:
Do you have a work item to add better documentation?

Reply:
Will be adding more documentation and code samples after this is merged.
Issue - https://github.com/azure/azure-sdk-for-java/issues/5545